packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -187,17 +199,24 @@ const DialogueMatchRow = ({ rowProps, classes, showMatchNote, onHide }: Dialogue
         </PostsItem2MetaInfo>
       </div>
       <PostsItem2MetaInfo className={classes.dialogueMatchNote}>
-        {showMatchNote ? "You've matched!" : "Check to opt in to dialogue, if you find a topic"}
+        {!matchPreference && (reciprocalMatchPreference ? "Waiting for you →" : "You've matched") }
```

Comment:
Is showing the text conditional on `matchPreference`, or showing the entire element wrapper?  I assume you don't want an empty wrapper with no text.

Reply:
ah yeah, thanks!